suite-native/module-send/src/components/ReviewOutputItem.tsx

```diff
@@ -0,0 +1,74 @@
+import { AlertBox, HStack, Text, VStack } from '@suite-native/atoms';
+import { ReviewOutputState } from '@suite-common/wallet-types';
+import { ReviewOutputType } from '@suite-common/wallet-types';
+import { CryptoAmountFormatter } from '@suite-native/formatters';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+
+import { StatefulReviewOutput } from '../types';
+
+const getAlertBoxVariant = (state: ReviewOutputState) => {
```

Comment:
nit: if you make ReviewOutputState defined for all option ( e.g. replace undefined with some string),  you can change this into object. Unless the optionality is used somehow.